pkg/logger/formatter.go

```diff
@@ -87,10 +87,12 @@ func (f *Formatter) Format(entry *logrus.Entry) ([]byte, error) {
 
 	if !f.DisableColor {
 		levelColor := getColorByLevel(entry.Level)
-		fmt.Fprintf(b, "\x1b[%dm%s\x1b[0m", levelColor, newLog)
+		fmt.Fprintf(b, "\033[%dm%s\033[0m", levelColor, newLog)
```

Comment:
Add a comment for this line, otherwise nobody could recognize what this line of code mean. @kakaZhou719

Reply:
I didn't modify the log colors, just added the format specification

here is  an color examples:

`var Reset  = "\033[0m"
var Red    = "\033[31m"
var Green  = "\033[32m"
var Yellow = "\033[33m"
var Blue   = "\033[34m"
var Purple = "\033[35m"
var Cyan   = "\033[36m"
var Gray   = "\033[37m"
var White  = "\033[97m"`